added STM32H750VBTx

wykys opened the pull request:
Added symbol for STM32H750VBTx

https://www.st.com/resource/en/datasheet/stm32h750ib.pdf

Checking symbol 'STM32H750VBTx':
  Violating S3.1
    > Symbol slightly off-center
    >   Center calculated @ (0, -50)

The symbol is shifted vertically by 50 miles, keeping the same boundary as the rest of the components in the library.

![STM32H750VBTx](https://user-images.githubusercontent.com/8275996/76167016-c1d77180-6163-11ea-9319-1e2944364b6b.png)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=2527) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=2527) before we can accept your contribution.<br/><hr/>**Jan Vykydal** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/KiCad/kicad-symbols?pullRequest=2527) it.</sub>

myfreescalewebpage commented:
Hello @wykys thanks for this symbol

Can you rebase your branch to permit the review ? Thanks.
Also, pin length should be 100mil and VSS pins should be stacked.
Will perform full review after that.

Joel

myfreescalewebpage commented:
@wykys ping

wykys commented:
Ahoj, zatím jsem udělal rebase a naskládal GND kolíky. Nemohu zkrátit délku na 100 mil, protože pin číslo 100 by se nehodilo na linii, mohu snížit délku kolíků na maximálně 150 mil.
Pokud je to v pořádku, mohu provést změny.

myfreescalewebpage commented:
Sorry it's a typo error, yes the pin length should be 150mil for this symbol.
Thanks.
Joel

myfreescalewebpage commented:
@wykys ping

wykys commented:
Deleted the STM32H750VBTx symbol and created a new one using an alias on STM32H753VITx.

I also changed PH0 and PH1 to bidirectional according to the datasheet.
Cite: [Reference manul: 11.3.14 Using the HSE or LSE oscillator pins as GPIOs ](https://www.st.com/resource/en/reference_manual/dm00314099-stm32h742-stm32h743-753-and-stm32h750-value-line-advanced-arm-based-32-bit-mcus-stmicroelectronics.pdf)
When the HSE or LSE oscillator is switched OFF (default state after reset), the related
oscillator pins can be used as normal GPIOs.
When the HSE or LSE oscillator is switched ON (by setting the HSEON or LSEON bit in the
RCC_CSR register) the oscillator takes control of its associated pins and the GPIO
configuration of these pins has no effect.
When the oscillator is configured in a user external clock mode, only the OSC_IN or
OSC32_IN pin is reserved for clock input and the OSC_OUT or OSC32_OUT pin can still be
used as normal GPIO.

myfreescalewebpage commented:
This is strange, the dcm file should be modified too! Can you fix ?
You have forced push so I can't understand why. Force pushing is not necessary, it really disturb the review so please do not use it.
Joel

myfreescalewebpage commented:
Now plenty of files modified, this is not expected... https://github.com/KiCad/kicad-symbols/pull/2527/files

wykys commented:
I tried to incorporate the comments. I'm ready to check.

myfreescalewebpage commented:
Back to previous status, dcm modification is missing

wykys commented:
I think the contents of the dcm file are correct:
https://github.com/wykys/kicad-symbols/blob/feature/STM32H750VBTx/MCU_ST_STM32H7.dcm#L45-L49

myfreescalewebpage commented:
@wykys ping, are you still working on this ?
